aws/testdata/mirrors_linux2023.txt

```diff
@@ -1 +1 @@
-https://cdn.amazonlinux.com/al2023/core/guids/46ff4933b89b948580f3b223b826fee3c1830b85885db3f7f90502c0ac99698c/x86_64/
+https://cdn.amazonlinux.com/al2023/core/guids/46ff4933b89b948580f3b223b826fee3c1830b85885db3f7f90502c0ac99698c/x86_64/
```

Comment:
This test case is updated to remove the trailing newline and better mimic the prod mirror.list